docs/2_4_common_schema.adoc

```diff
@@ -1711,7 +1711,7 @@ Knowns latexmath:[{\mathbf{v}_{\mathit{known}}}] in <<InitializationMode>> (for
 * <<independent>> variable (usually time; <<causality>> = <<independent>>).
 
 _[The <<independent>> variable should be treated as any other variable when it comes to dependencies._
-_As only explicit dependencies are listed it is required that unknowns that depends on the <<independent>> variable have a partial derivative that is different from zero._
+_As only explicit dependencies are listed it is required that unknowns that depend on the <<independent>> variable have a partial derivative w.r.t this variable that is different from identically zero._
```

Comment:
# #`_As only explicit dependencies are listed, it is required that unknowns that depend on the <<independent>> variable have a partial derivative w.r.t. this variable that is different from identically zero._`